frappe/installer.py

```diff
@@ -128,9 +130,10 @@ def install_db(
 	db_host=None,
 	db_port=None,
 	no_mariadb_socket=False,
+	setup=True,
 ):
 	import frappe.database
-	from frappe.database import setup_database
+	from frappe.database import bootstrap_database, setup_database
```

Comment:
This import seems to be broken, `setup_database` worked like that since it is separately defined in the database package: https://github.com/frappe/frappe/blob/develop/frappe/database/__init__.py#L10

You'd need a similar equivalent of `bootstrap_database` defined as well

Reply:
here we go: https://github.com/frappe/frappe/pull/23170/commits/afc547b56aaae569c28d077cac2f8fd9209cab00